packages/console/src/pages/Welcome/index.module.scss

```diff
@@ -43,12 +39,20 @@
 
 main {
   flex: 1;
+  display: flex;
+  flex-direction: column;
+}
+
+.placeHolderTop {
+  flex: 1;
+}
+
+.placeHolderBottom {
```

Comment:
use placeholder as one word should be ok

Reply:
updated